docs/source/guide/custom_metric.md

```diff
@@ -282,3 +283,4 @@ To grant permissions to a specific user, role, or EKS node group used to manage
    ]
 }
 ```
+</div>
```

Comment:
Read and insert <div class="enterprise-only">or <div class="opensource-only"> .
Important note: before and after div should be newlines, like on the screenshot!

Reference: https://heartex.slack.com/archives/C01HGFXA7KR/p1670507767449669